### ISSUE TITLE
App crushes after root node is selected

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/fta-fmea-ui#331]
_Originally posted by @blcham in https://github.com/kbss-cvut/fta-fmea-ui/pull/330#discussion_r1620642735_

As workaround @Kasmadei  suggested to change:
`src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx:100`

as
```
  return (
    <Box paddingLeft={2} marginRight={2}>
      {/* Hidden to prevent app crush after root node select (Bug with UI). Will be fixed in next PR */}
      {/* <FaultEventShapeToolPane data={shapeToolData} onEventUpdated={onEventUpdated} refreshTree={refreshTree} /> */}

      {/* TODO: Finish for other nodes. Will be refactored. */}
```

**Comment by blcham** [comment]
@Kasmadei please add additional description and related PR to fix unless you have another suggestion

**Comment by blcham** [comment]
@LaChope please create workaround PR and merge if it works while leaving this ISSUE open

**Comment by blcham** [comment]
Closing this issue event it is not resolved as it is duplicate for #316 .
